raiden/tests/utils/events.py

```diff
@@ -78,6 +34,82 @@ def check_dict_nested_attrs(item, dict_data):
     return True
 
 
+def check_nested_attrs(item: Any, attributes: dict) -> bool:
```

Comment:
I think we try to stick with the `typing` types. So `typing.Dict` and `typing.Bool`.

Same for the other functions below, with `List` e.t.c.

Reply:
It's a styling decision. I know there is no real difference. But I would like us to be consistent.

Sorry for the bool -- disregard it. But for the rest's let's be consistent.